Backend/src/constants/error.constants.ts

```diff
@@ -0,0 +1,9 @@
+export const ErrorCode = {
+  NoResponseFromServer: 'j-001',
+  ServerError: 'j-002',
+};
+
+export const ErrorMsg = {
+  NoResponseFromServer: 'No response from server',
```

Comment:
We need to be more specific here. Trying to connect to server on IP on this port, and then say No response from server. That is more meaningful.

Reply:
changed

Reply:
Ok